lib/lita-lean_tc.rb

```diff
@@ -62,6 +66,15 @@ def build_finished(payload)
           create_confirmed
         end
       end
+      # Returns a list of all cards on the Feature request board
+      #
+      def list_feature_request(response)
```

Comment:
An excellent motivator. I've moved the logic into a new method in the `FeatureRequests` class.